### DISCUSSION TITLE
Why can't I persist data?

### DISCUSSION THREAD

**Discussion by bogdanstate** [PrefectHQ/prefect#3187] [Q&A] [answered]
I am quite impressed with Prefect, currently playing with it for a project using the Prefect server. I am just starting out so I am not sure if this is a bug or a problem with my understanding of the data model. 

So here's what I'm doing:
1. Starting a new prefect server (installed today using conda), and agents, with a dask cluster.
2. Go to GUI, create a new project, let's say "test".
3. Project is successfully created.
4. Trying to stop the server gracefully, by pressing `Ctrl+C` in the server Terminal window. Then I get the following reply:
```
^CGracefully stopping... (press Ctrl+C again to force)
Exception caught; killing services (press ctrl-C to force)
Stopping tmp_ui_1       ... done
Stopping tmp_apollo_1   ... 
Stopping tmp_towel_1    ... 
Stopping tmp_graphql_1  ... 
Stopping tmp_hasura_1   ... 
Stopping tmp_postgres_1 ... 
Stopping tmp_apollo_1   ... done
Stopping tmp_towel_1    ... done
Stopping tmp_graphql_1  ... done
Stopping tmp_hasura_1   ... done
Stopping tmp_postgres_1 ... done
Removing tmp_ui_1       ... done
Removing tmp_apollo_1   ... done
Removing tmp_towel_1    ... done
Removing tmp_graphql_1  ... done
Removing tmp_hasura_1   ... done
Removing tmp_postgres_1 ... done
Removing network prefect-server

Aborted!
```
 5. Restarting the server using `prefect server start`

My project, and all the state of the Postgres DB is gone. I have a clean slate. This looks less than ideal.

There's a ton of logs that I am getting from the server, so not quite sure what I should look at to diagnose this later.

**Comment by joshmeek** [comment, marked as answer]
Hi @bogdanstate the server does not persist data by default and you will have to enable it with a couple extra CLI flags. More information here: https://docs.prefect.io/orchestration/server/deploy-local.html#database-persistence-and-migrations

**Reply by bogdanstate** [reply]
Thanks, this looks like what I was looking for!
